src/get-started/java-binding.md

```diff
@@ -1375,6 +1375,35 @@ YAML dictionary. The Instance Configuration plugin also requires a list of
 additional parameters. In the present example the parameters refer to the
```

Comment:
Would it make sense to rewrite the sentence "The Instance Configuration plugin also requires a list of
additional parameters" to be more concrete, as these are not additional, but one of the primary parameters:

"The Instance Configuration plugin supports several configuration formats: 
* some standard text formats: JSON, properties, plain text;
* protocol buffers."

and then define two sub-sections:

- Text Configuration Formats

- Protocol Buffers Configuration Format (already added)

Reply:
I did it, please take a look